core/services/versionchooser/static/index.html

```diff
@@ -286,6 +287,22 @@ <h3>Applying...</h3>
             },
           }).finally(() => setInterval(this.checkIfBackendIsOnline, 1000))
         },
+        deleteVersion (fullname) {
+          const [repository, tag] = fullname.split(':')
+          axios({
+            method: 'delete',
+            url: '/v1.0/version/delete',
+            data: {
+              repository,
+              tag,
+            },
+          }).then(() => {
+            this.availableVersions['local'] = this.availableVersions['local'].filter(
```

Comment:
isn't it better to call `loadAvailableversions()` ? Or is there a specific reason to update `availableVersions` ? If so, could you comment it ?

Reply:
loadAvailableVersions takes a while, fetching from dockerhub can take some time. this is instantaneous, I'll add a comment.